### ISSUE TITLE
Non-crafting way of getting ender pearls

### ISSUE THREAD

**Issue by PizzaPolice** [reuy/Freelancer-Initiative#88]
Farming endermen just sucks + needed for a lot of mid-high tier recipes, must be expensive enough to not be spammed but enough that you will won't feel terrible for having to make them

**Comment by reuy** [comment]
Agreed, especially because endermen are rarer due to spawn competition. This might be a magister recipe
